lerobot/common/datasets/lerobot_dataset.py

```diff
@@ -291,7 +291,8 @@ def create(
         obj.root.mkdir(parents=True, exist_ok=False)
 
         if robot is not None:
-            features = get_features_from_robot(robot, use_videos)
+            features = {} if features is None else features
+            features.update(get_features_from_robot(robot, use_videos))
```

Comment:
This comment was also made on #518 . 
What do you think of writing it like this?
```suggestion
            features = {**(features or {}), **get_features_from_robot(robot)}
```